.eslintrc

```diff
@@ -1,14 +1,41 @@
 {
   "parser": "babel-eslint",
   "rules": {
-      "max-len": [1, 120, 2, {"ignoreComments": true }],
-      "prop-types": [2],
-      "import/no-extraneous-dependencies": ["error", {"devDependencies": true}]
+    "max-len": [
+      1,
+      120,
+      2,
+      {
+        "ignoreComments": true,
+        "ignoreUrls": true
+      }
+    ],
+    "react/prop-types": [
+      2
+    ],
+    "import/no-extraneous-dependencies": [
+      "error",
+      {
+        "devDependencies": true
+      }
+    ],
+    "global-require": 0,
+    "no-console": "off",
```

Comment:
I'd suggest leaving this rule enabled for client app (remove all console statements), and disabled for server side (it's ok to have console statement on server) - https://eslint.org/docs/rules/no-console